kotlin.js

```diff
@@ -0,0 +1,252 @@
+import fs from "fs"
+import path from "path"
+import assert from "assert"
+import process from "process"
+import fsPromises from "fs/promises"
+
+import core from "@actions/core"
+
+import Tool from "./tool.js"
+
+export default class Kotlin extends Tool {
```

Comment:
It seems this tool and java are very similar (as they are both managed via sdman). Maybe it's worth having an abstract class or delegate class that does all the sdkman stuff and the kotlin and java tools just invoke it

Reply:
Agreed, having something like `SdkmanTool` that extends `Tool` from which `Java` and `Kotlin` would then extend `SdkmanTool` makes good sense, have done this in Java a million times in the past.